ground/src/main/res/layout/map_type_dialog_item.xml

```diff
@@ -25,15 +25,18 @@
 
   <ImageView
     android:id="@+id/imageview"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
-    tools:src="@drawable/ground_logo" />
+    android:layout_width="128dp"
+    android:layout_height="128dp"
+    android:padding="7dp"
+    tools:src="@drawable/ic_type_roadmap" />
```

Comment:
Layout nit: Can we align the left and right sides of the images with the "Map type" header and close button?